### PR TITLE
Reattach

### LINKED ISSUE #18: As a User I should be able to click on a favorite card and see details about the city
The details page should be similar to the Main page view with a few tweaks:
- Image
- Name
- Summary
- quality of life infographic
- NO magic 8 ball
- NO "should I move button"
- NO X or <3 in the tap bar, only the user icon

### PR DESCRIPTION
**Pull request by clairefields15**
# Description of Changes/Motivation
Added a details page that you can access by clicking on a favorites card. It has all the same information as the main page except no 8 ball and no like/dislike.
Added navigation to favorites and home from that page.
Did some general styling and refactoring across all files.

## Issues Closed
closes #18 

## Type of change

Check the relevant option:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor (clean up code)

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -64,19 +65,19 @@ export const App = () => {
   }, [allCities]);
 
   useEffect(() => {
-    if (cityImage) {
+    if (Object.keys(cityImage).length !== 0) {
```
Comment (ashton-huxtable) [inline src/components/App/App.js]: I think you could leave off the !== 0 here and achieve the same result!

File: src/components/ErrorComponent/ErrorComponent.js [hunk 2]
```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import './ErrorComponent.css';
 
 export const ErrorComponent = ({ errorMessage }) => {
   return (
-    <>
-      <h2>{errorMessage}</h2>
+    <div className='error-page'>
```
Comment (ashton-huxtable) [inline src/components/ErrorComponent/ErrorComponent.js]: Would this be better as a semantic tag like <section>?

**Review by ashton-huxtable — APPROVED** [review]
Really great, clean code - Only a couple of small nit[icky suggestions that are more of a personal preference!

File: src/components/App/App.js (hunk 1 above)
Comment (clairefields15) [reply src/components/App/App.js]: Oh yeah! Nice!

File: src/components/ErrorComponent/ErrorComponent.js (hunk 2 above)
Comment (clairefields15) [reply src/components/ErrorComponent/ErrorComponent.js]: Mmm I love section soup

**Pull request merged**